Go/Snippets/go-ifelse.sublime-snippet

```diff
@@ -0,0 +1,10 @@
+<snippet>
+    <content><![CDATA[if $0 {
+
```

Comment:
Maybe have placeholders within `if-else` blocks so users could directly place content in that ?